src/zkevm_specs/evm/execution/create.py

```diff
@@ -0,0 +1,149 @@
+    instruction.transfer(caller_address, contract_address, value)
+
+    # gas cost of memory expansion
+    (next_memory_size, memory_expansion_gas_cost,) = instruction.memory_expansion_constant_length(
+        offset,
+        size,
+    )
+
+    # gas cost of CREATE = GAS_COST_CREATE + memory expansion + GAS_COST_CODE_DEPOSIT * len(byte_code)
+    # the last one (GAS_COST_CODE_DEPOSIT * len(byte_code)) would be calculated in `return_revert`
+    gas_left = instruction.curr.gas_left
+    gas_cost = GAS_COST_CREATE + memory_expansion_gas_cost
+    gas_available = gas_left - gas_cost
+
+    # Apply EIP 150
+    one_64th_gas, _ = instruction.constant_divmod(gas_available, FQ(64), N_BYTES_GAS)
+    all_but_one_64th_gas = gas_available - one_64th_gas
+    is_u64_gas = instruction.is_zero(
+        instruction.sum(instruction.rlc_encode(gas_left).le_bytes[N_BYTES_GAS:])
+    )
+    callee_gas_left = instruction.select(
+        is_u64_gas,
+        instruction.min(all_but_one_64th_gas, gas_left, N_BYTES_GAS),
+        all_but_one_64th_gas,
+    )
+
+    # calculate code_hash = hash(init_code)
+    copy_rwc_inc, rlc_acc = instruction.copy_lookup(
+        instruction.curr.call_id,  # src_id
+        CopyDataTypeTag.Memory,  # src_type
+        code_hash,  # dst_id
+        CopyDataTypeTag.Bytecode,  # dst_type
+        offset,  # src_addr
+        offset + size,  # src_addr_boundary
+        FQ(0),  # dst_addr
+        size,  # length
+        instruction.curr.rw_counter + instruction.rw_counter_offset,
+    )
+    code_hash = instruction.keccak_lookup(size, rlc_acc)
+
+    stack_pointer_delta = 2 + is_create2
+    # Save caller's call state
+    for field_tag, expected_value in [
+        (CallContextFieldTag.ProgramCounter, instruction.curr.program_counter + 1),
+        (
+            CallContextFieldTag.StackPointer,
+            instruction.curr.stack_pointer + stack_pointer_delta,
+        ),
+        (CallContextFieldTag.GasLeft, gas_left - gas_cost - callee_gas_left),
+        (CallContextFieldTag.MemorySize, next_memory_size),
+        (
+            CallContextFieldTag.ReversibleWriteCounter,
+            instruction.curr.reversible_write_counter + 1,
+        ),
+    ]:
+        instruction.constrain_equal(
+            instruction.call_context_lookup(field_tag, RW.Write),
+            expected_value,
+        )
+
+    # Setup next call's context.
+    for field_tag, expected_value in [
+        (CallContextFieldTag.CallerId, instruction.curr.call_id),
+        (CallContextFieldTag.TxId, tx_id.expr()),
+        (CallContextFieldTag.Depth, depth.expr() + 1),
+        (CallContextFieldTag.CallerAddress, caller_address.expr()),
+        (CallContextFieldTag.CalleeAddress, contract_address.expr()),
+        (CallContextFieldTag.IsSuccess, FQ(True)),
+        (CallContextFieldTag.IsStatic, FQ(False)),
+        (CallContextFieldTag.IsRoot, FQ(False)),
+        (CallContextFieldTag.IsCreate, FQ(False)),
```

Comment:
```suggestion
        (CallContextFieldTag.IsCreate, FQ(True)),
```

Reply:
fixed in [80d39e8](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/355/commits/80d39e83047d4084c7c9b29ae51f9c9c0b1a9d42)